openpype/hosts/nuke/api/workio.py

```diff
@@ -1,6 +1,7 @@
 """Host API required Work Files tool"""
 import os
 import nuke
+from openpype.hosts.nuke.api.lib import is_headless
```

Comment:
You've now introduced recurive imports. `workio` imports `lib` and `lib` imports `workio`. I assume it's currently broken.

Reply:
Tested it just now. it is broken. maybe it is better to keep it in workio.py as the original. i.e. `headless = QtWidgets.QApplication.instance() is None`